rclcpp/include/rclcpp/context.hpp

```diff
@@ -318,7 +323,6 @@ class Context : public std::enable_shared_from_this<Context>
 
   /// Interrupt any blocking sleep_for calls, causing them to return immediately and return true.
   RCLCPP_PUBLIC
-  virtual
   void
   interrupt_all_sleep_for();
```

Comment:
is this related to the PR description?
I don't see directly how.

Reply:
These functions were called from the destructor and therefore cannot be virtual.